### ISSUE TITLE
race condition in constant arrival rate

### ISSUE THREAD

**Issue by MStoykov** [loadimpact/k6#1477]
```
==================
WARNING: DATA RACE
Write at 0x00c000e1ffc0 by goroutine 59:
  github.com/loadimpact/k6/lib/executor.ConstantArrivalRate.Run()
      /home/mstoykov/.gvm/pkgsets/go1.14.2/global/src/github.com/loadimpact/k6/lib/executor/constant_arrival_rate.go:293 +0x1778
  github.com/loadimpact/k6/lib/executor.(*ConstantArrivalRate).Run()
      <autogenerated>:1 +0x10d
  github.com/loadimpact/k6/core/local.(*ExecutionScheduler).runExecutor()
      /home/mstoykov/.gvm/pkgsets/go1.14.2/global/src/github.com/loadimpact/k6/core/local/local.go:325 +0x49e

Previous read at 0x00c000e1ffc0 by goroutine 89:
  github.com/loadimpact/k6/lib/executor.ConstantArrivalRate.Run.func4()
      /home/mstoykov/.gvm/pkgsets/go1.14.2/global/src/github.com/loadimpact/k6/lib/executor/constant_arrival_rate.go:264 +0xbe
  github.com/loadimpact/k6/lib/executor.trackProgress()
      /home/mstoykov/.gvm/pkgsets/go1.14.2/global/src/github.com/loadimpact/k6/lib/executor/helpers.go:174 +0x157

Goroutine 59 (running) created at:
```
This happened while running all tests with --race 
As far as I can see from TestExecutionSchedulerRunCustomTags/constant-arrival-rate but I would guess this really doesn't matter all that much :D

**Comment by na--** [comment]
Hmm this seems superfluous? https://github.com/loadimpact/k6/blob/83d0fcf2ecf44db27275cdf7ea887c05edf9bd06/lib/executor/constant_arrival_rate.go#L293

**Comment by MStoykov** [comment]
Hm ... yeah, I have probably added it to increase the accuracy ... but this might actually do the opposite ...

**Comment by MStoykov** [comment]
fixed with 33d65e94b65ed40639d388cc1ff09308a2299d51
